superdesk.config.js

```diff
@@ -16,6 +16,11 @@ module.exports = function(grunt) {
                 subscriptions: true,
             },
             validatePointOfInterestForImages: true,
+            customAuthoringTopbar: {
```

Comment:
why changing this?

Reply:
Sorry, my bad! Left over on testing some other issue. Removing...